core/src/dunit/scala/io/snappydata/cluster/DeployUndeployDUnitTest.scala

```diff
@@ -93,58 +104,117 @@ class CassandraSnappyDUnitTest(val s: String)
     new File(snappyProductDir, "books.xml").createNewFile()
     sparkXmlJarPath = downloadURI("https://repo1.maven.org/maven2/com/databricks/" +
         "spark-xml_2.11/0.5.0/spark-xml_2.11-0.5.0.jar")
-    val cassandraJarLoc = getLoc(downloadLoc)
-    cassandraConnectorJarLoc =
-      getUserAppJarLocation("spark-cassandra-connector_2.11-2.0.7.jar", downloadLoc)
-    if (cassandraJarLoc.nonEmpty && cassandraConnectorJarLoc != null) {
-      cassandraClusterLoc = cassandraJarLoc.head
-    } else {
-      ("curl -OL http://www-us.apache.org/dist/cassandra/" +
-          s"2.1.21/apache-cassandra-2.1.21-bin.tar.gz").!!
-      ("curl -OL https://repo1.maven.org/maven2/com/datastax/spark/" +
-          "spark-cassandra-connector_2.11/2.0.7/" +
-          "spark-cassandra-connector_2.11-2.0.7.jar").!!
-      val jarLoc = getUserAppJarLocation("apache-cassandra-2.1.21-bin.tar.gz", currDir)
-      val connectorJarLoc =
-        getUserAppJarLocation("spark-cassandra-connector_2.11-2.0.7.jar", currDir)
-      ("tar xvf " + jarLoc).!!
-      val loc = getLoc(currDir).head
-      if (downloadLoc.nonEmpty) {
-        s"rm -rf $downloadLoc/*"
-      }
-      s"cp -r $loc $downloadLoc".!!
-      s"mv $connectorJarLoc $downloadLoc".!!
-      cassandraClusterLoc = s"$downloadLoc/apache-cassandra-2.1.21"
-      cassandraConnectorJarLoc = s"$downloadLoc/spark-cassandra-connector_2.11-2.0.7.jar"
-    }
-    logInfo("CassandraClusterLocation : " + cassandraClusterLoc +
-        " CassandraConnectorJarLoc : " + cassandraConnectorJarLoc)
-    (cassandraClusterLoc + "/bin/cassandra").!!
-    logInfo("Cassandra cluster started")
+
+    // start cassandra cluster
+    jarLoc = getLoc(downloadLoc, "apache-cassandra-2.1.21")
+    connectorJarLoc =
+        getUserAppJarLocation("spark-cassandra-connector_2.11-2.0.7.jar", downloadLoc)
+    jarUrl = "http://www-us.apache.org/dist/cassandra/2.1.21/apache-cassandra-2.1.21-bin.tar.gz"
+    connectorJarUrl = "https://repo1.maven.org/maven2/com/datastax/spark/" +
+        "spark-cassandra-connector_2.11/2.0.7/spark-cassandra-connector_2.11-2.0.7.jar"
+    startCluster("apache-cassandra-2.1.21-bin.tar.gz",
+      "spark-cassandra-connector_2.11-2.0.7.jar",
+      jarUrl, connectorJarUrl, "apache-cassandra-2.1.21")
+    cassandraClusterLoc = clusterLoc
+    cassandraConnectorJarLoc = connectorJarLoc
+    startCassandraCluster()
+
+    // start mongodb cluster
```

Comment:
So this is all that is needed to run monogodb, right? Please ensure anyone running precheckin should not see a failure here for lack of any additional setup.

Reply:
Ok, will do